app-office/texstudio/texstudio-2.12.16.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="LaTeX made comfortable"
+DESCRIPTION="TeXstudio is an integrated writing environment for creating \
+LaTeX documents. Our goal is to make writing LaTeX as easy and comfortable as \
+possible. Therefore TeXstudio has numerous features like syntax-highlighting, \
+integrated viewer, reference checking and various assistants. "
+HOMEPAGE="https://www.texstudio.org/"
+COPYRIGHT="Benito van der Zander"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/texstudio-org/texstudio/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="a14b8912bfd15d982cfbe5f00deed37ca85fb6e38d3aa0c2dac23b4ecaab0984"
+PATCHES="texstudio-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+PROVIDES="
+	texstudio$secondaryArchSuffix = $portVersion
+	cmd:texstudio = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+#	lib:libpoppler$secondaryArchSuffix
```

Comment:
this can be removed.

Reply:
Hence it was disabled for checking :) (as the next one)
We could use system hunspell and quazip, but it's hard to check here due to the project size/compilation time.